### PR TITLE
add support for plain text

### PR DESCRIPTION
**Pull request by wentingliu**

### CODE REVIEW

File: lib/text.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  opts = opts || {};
+
+  // defaults
+  var len = req.headers['content-length'];
+  if (len) opts.length = ~~len;
+  opts.encoding = opts.encoding || 'utf8';
+  opts.limit = opts.limit || '1mb';
+
+  return function(done){
+    raw(req, opts, function(err, str){
```
Comment (fengmk2) [inline lib/text.js]: @wentingliu can you fixes these?

**Comment by fengmk2** [comment]
Landed 021efe49519a319702de346e87d391f0253c2018

**Comment by fengmk2** [comment]
Thanks @wentingliu

**Comment by fengmk2** [comment]
@coderhaoxin @wentingliu co-body@1.1.0 published

**Comment by coderhaoxin** [comment]
:tada:

**Pull request closed without merging**